custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -346,36 +346,44 @@ function ServiceDeliveryDashboardController($rootScope, $scope, $http, $location
     };
 
     // mobile helpers
+    const NO_TOOLTIP_DISPLAYED = -1;
+    const DEFAULT_SORTED_COLUMN = 0;
+    const DEFAULT_SORTING_DIRECTION = 0;
```

Comment:
sorry i should have been a little more specific. one constant I had in mind, but didn't mention explicitly was having 
`SORT_ASCENDING = 0` and `SORT_DESCENDING = 1` which would improve readability. though is not important and maybe you can do it in a follow up so we can get this in first if you agree